models/comments.model.js

```diff
@@ -37,5 +37,27 @@ const insertCommentByArticleId = (comment) => {
     return result.rows[0];
   });
 };
+const fetchCommentByCommentId = (comment_id) => {
+  if (!/^\d+$/.test(comment_id)) {
+    throw new BadRequestError();
+  }
 
-module.exports = { fetchCommentsByArticleId, insertCommentByArticleId };
\ No newline at end of file
+  return db.query('SELECT * FROM comments WHERE comment_id = $1;',
+    [comment_id]).then((result) => {
+    if (!result.rows[0]) {
+      throw new NotFoundError('Comment does not exist');
+    }
+    return result.rows[0];
+  });
+};
+const deleteCommentByIdFromDB = (comment_id) => {
+
+  return fetchCommentByCommentId(comment_id).then(() => {
+    db.query('DELETE FROM comments WHERE comment_id = $1;',
+      [comment_id]).then((result) => {
+    });
+  });
+  
```

Comment:
You could do the 404 check with just one database call, by checking the rowCount on the results